src/features/forgotPwd/index.js

```diff
@@ -0,0 +1,61 @@
+import React, { useRef, useState } from "react";
+import { Form, Button, Card, Alert, Container } from "react-bootstrap";
+import useAuth from "../../hooks/useAuth";
+import { Link } from "react-router-dom";
+
+const ForgotPassword = () => {
+  const emailRef = useRef();
+  const { resetPassword } = useAuth();
+  const [error, setError] = useState("");
+  const [message, setMessage] = useState("");
+  const [loading, setLoading] = useState(false);
+
+  async function handleSubmit(e) {
+    e.preventDefault();
+
+    try {
+      setMessage("");
+      setError("");
+      setLoading(true);
+      await resetPassword(emailRef.current.value);
+      setMessage("Check your inbox for further instructions");
+    } catch {
+      setError("Failed to reset password");
+    }
+
+    setLoading(false);
+  }
+
+  return (
+    <Container
+      className="d-flex align-items-center justify-content-center"
+      style={{ minHeight: "100vh" }}
+    >
+      <div className="w-100" style={{ maxWidth: "400px" }}>
+        <Card>
+          <Card.Body>
+            <h2 className="text-center mb-4">Password Reset</h2>
+            {error ? <Alert variant="danger">{error}</Alert> : null}
+            {message && <Alert variant="success">{message}</Alert>}
```

Comment:
I would use this `{message && <Alert variant="success">{message}</Alert>}`
as `{message ? <Alert variant="success">{message}</Alert> : null}`

Reply:
sir hisham told me to do this way.